app/views/users/_edit.haml

```diff
@@ -71,7 +71,7 @@
         %h3= t(".change_color_theme")
         = form_for "user", url: edit_user_path, html: {method: :put} do |f|
           .form-inline.clearfix
-            = f.select :color_theme, available_color_themes, {}, class: "form-control form-group"
+            = f.select :color_theme, available_color_themes, {}, class: "form-control form-group color-theme"
```

Comment:
There is still a leftover class here.

Reply:
Removed, thanks for being such an eagle-eye.